README.md

```diff
@@ -54,27 +54,26 @@ The following figure shows the typical CoatRack architecture, the CoatRack web a
 
 ## Install
 
-Scripts for building and running CoatRack are available in the root folder of the code base. To build CoatRack, use:
+Go to the docker-compose-setup folder and execute:
 
-```console
-./build.sh
-```
-To run the CoatRack web application, use:
+ ```sh
+ sh init.sh
```

Comment:
"init" sounds to me like it only sets some local variables, path config etc.

Maybe a more descriptive name should be used, e.g. `build-and-init-docker-volumes.sh` or `init-local-docker-compose-setup.sh` or `install-dockerized-coatrack.sh`?

Reply:
or maybe simply `install.sh`?